tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw

```diff
@@ -161,9 +161,9 @@
     <value>Enable end task in taskbar by right click</value>
     <comment>The description for the end task on task bar settings card</comment>
   </data>
-  <data name="ExampleDevDriveLocation" xml:space="preserve">
-    <value>Example: E:\packages\pip</value>
-    <comment>Example dev drive location</comment>
+  <data name="ExampleText" xml:space="preserve">
+    <value>Example: </value>
```

Comment:
Don't include space here, I'm not sure it gets translated.

Reply:
It is getting reflected.

Reply:
I was referring to the files that aren't checked in, the ones translated to other languages. But it's probably fine.